app.py

```diff
@@ -48,6 +49,7 @@ def _fk_pragma_on_connect(dbapi_con, con_record):  # noqa
     with flask_app.app_context():
         event.listen(db.engine, "connect", _fk_pragma_on_connect)
         db.create_all()
+        seed_db()  # TODO: Remove this after pen test
```

Comment:
Sorry for change of heart, can we model this as a [Flask Custom Command](https://flask.palletsprojects.com/en/2.3.x/cli/#custom-commands) as well, that way it will be easier to run once we connect up to Postgres and we need to run it one time.

Reply:
nice idea - implemented 👍